nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/groups/StandardProcessGroup.java

```diff
@@ -4894,7 +4894,7 @@ public void verifyCanUpdate(final VersionedFlowSnapshot updatedFlow, final boole
                 // any connection that does exist in the proposed flow. This results in us having a Map whose values are those Connections
                 // that were removed. We can then check for any connections that have data in them. If any Connection is to be removed but
                 // has data, then we should throw an IllegalStateException.
-                findAllConnections().forEach(conn -> removedConnectionByVersionedId.put(conn.getVersionedComponentId().orElse(conn.getIdentifier()), conn));
+                flowManager.findAllConnections().forEach(conn -> removedConnectionByVersionedId.put(conn.getVersionedComponentId().orElse(conn.getIdentifier()), conn));
```

Comment:
I think this call is actually changing the logic. `findAllConnections()` here is going to return all Connections in this ProcessGroup and any children/descendant groups. This changes it from "all connections here and below" to "all connections in the canvas," which I don't think is what we want.